mlir-sycl/test/Conversion/SYCLToLLVM/sycl-call-to-llvm.mlir

```diff
@@ -1,5 +1,20 @@
 // RUN: sycl-mlir-opt -split-input-file -convert-sycl-to-llvm -verify-diagnostics %s | FileCheck %s
 
+//===-------------------------------------------------------------------------------------------------===//
+// sycl.call with non void return type
+//===-------------------------------------------------------------------------------------------------===//
+
+// CHECK: llvm.func @foo() -> [[MEMREF:!llvm.struct<\(ptr<i32>, ptr<i32>, i64, array<1 x i64>, array<1 x i64>\)>]]
+// CHECK: llvm.func @test() -> !llvm.struct<(ptr<i32>, ptr<i32>, i64, array<1 x i64>, array<1 x i64>)> {
```

Comment:
Can use [[MEMREF]]:
```
llvm.func @test() -> [[MEMREF]] {

Reply:
Actually the name MEMREF is a confusing because the matched pattern is a struct not a memref. USe STRUCT_TYPE ?